<?xml version="1.0"?>
<launch>

	<node pkg="object_reaching" type="main" name='ee_control' respawn='false' output='screen'>
		<rosparam command="load" file="$(find object_reaching)/config/params.yaml"/>
	</node>
	
</launch>